src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <li
+        v-bind:key="props.id"
```

Comment:
why you add key attreibute? This element is useful when you render elements in loop.